content/v2.0/monitor-alert/templates/aws.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Monitor Amazon Web Services (AWS)
+description: >
+  Use the AWS CloudWatch Monitoring template to monitor data from Amazon Web Services (AWS), Amazon Elastic Compute Cloud (EC2), and Amazon Elastic Load Balancing (ELB) with the AWS CloudWatch Service.
+menu:
+  v2_0:
+    parent: Monitor with templates
+    name: AWS CloudWatch
+weight: 201
+---
+
+Use the [AWS CloudWatch Monitoring template](https://github.com/influxdata/community-templates/tree/master/aws_cloudwatch) to monitor data from [Amazon Web Services (AWS)](https://aws.amazon.com/), [Amazon Elastic Compute Cloud (EC2)](https://aws.amazon.com/ec2/), and [Amazon Elastic Load Balancing (ELB)](https://aws.amazon.com/elasticloadbalancing/) with the [AWS CloudWatch Service](https://aws.amazon.com/cloudwatch/).
+
+The AWS CloudWatch Monitoring template includes the following:
+
+- two dashboards:
+  - **AWS CloudWatch NLB (Network Load Balancers) Monitoring**: Displays data from the `cloudwatch_aws_network_elb measurement`
+  - **AWS CloudWatch Instance Monitoring**: Displays data from the `cloudwatch_aws_ec2` measurement
+- two labels: `inputs.cloudwatch`, `AWS`
+- one variable: `v.bucket`
+- one Telegraf configuration: [AWS CloudWatch input plugin](/v2.0/reference/telegraf-plugins/#cloudwatch)
+
+## Apply the template
+
+1. Use the [`influx` CLI](/v2.0/reference/cli/influx/) to run the following command:
+
+    ```sh
+    influx apply -f https://raw.githubusercontent.com/influxdata/community-templates/master/aws_cloudwatch/aws_cloudwatch.yml
+    ```
+    For more information, see [influx apply](/v2.0/reference/cli/influx/apply/).
+2. [Install Telegraf](/telegraf/latest/introduction/installation/) on a server with network access to both the CloudWatch API and [InfluxDB v2 API](/v2.0/reference/api/).
+3. In your Telegraf configuration file (`telegraf.conf`), find the following example `influxdb_v2` output plugins, and then **replace** the `urls` to specify the servers to monitor:
+
+   ```sh
+    ## k8s
+    [[outputs.influxdb_v2]]
+     urls = ["http://influxdb.monitoring:9999"]
+     organization = "InfluxData"
+     bucket = "kubernetes"
+     token = "secret-token"
+ 
+    ## cloudv2 sample
+    [[outputs.influxdb_v2]]
+     urls = ["$INFLUX_HOST"]
+     token = "$INFLUX_TOKEN"
+     organization = "$INFLUX_ORG"
+     bucket = “cloudwatch"
+   ```
```

Comment:
@mhall119, a couple questions perhaps you can help with:
- Will the template still work if folks alter these two bucket names? 
- Is Kubernetes "k8s" output also an "example" in this context? 
- Why would folks use one bucket or the other? Is providing 2 examples simply showing how to alternatively set url as $INFLUX_HOST?

Reply:
1. It looks like it would still work, since all the Flux scripts use the `v.bucket` variable, and the variable is populated by a Flux query that gets all available buckets, the user will just need to have created whatever buckets they're going to tell Telegraf to use

1. As it is, Telegraf will try to send measurements to two different InfluxDB instances, each one having a different bucket name. I don't know if the intention was for the user to pick one and delete the other, or use both simultaneously, you would have to ask @influxdata/bonitoo about that

1. Another question for @influxdata/bonitoo, but looking at the template the user wouldn't necessarily choose between one bucket or the other, they would use whichever bucket name was being used for the InfluxDB instance they were looking at (because again, there are two InfluxDB instances that Telegraf is sending measurements to)

Reply:
ad 1/ k8s and aws dashboards are indepedent templates. Both can share 1 bucket or can write to different buckets. No collissions in measurements should be observed as the aws and k8s are different sets of measuremensts.
ad 2/ The output bucket used is just an example
ad 3/ Maybe the bucket called "demo" (in the original k8s templaet) is not a good name for the default bucket and could be changed to kubernetes, instead